### PR TITLE
docs: Update pure docker deployment version to 3.35.0

### PR DESCRIPTION
**Pull request by kevinwojo**

### CODE REVIEW

File: doc/admin/updates/pure_docker.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ Each section comprehensively describes the changes needed in Docker images, envi
 
 <!-- GENERATE UPGRADE GUIDE ON RELEASE (release tooling uses this to add entries) -->
 
+## 3.34.2 -> 3.35
+
+To upgrade, please perform the changes in the following diff:
```
Comment (caugustus-sourcegraph) [inline doc/admin/updates/pure_docker.md:13]: Query-runner was deleted in this deployment - maybe worth noting.

Comment (caugustus-sourcegraph) [reply doc/admin/updates/pure_docker.md:13]: Oh! My brain caught up - doesn't that mean the diff is actually wrong?

Comment (kevinwojo) [reply doc/admin/updates/pure_docker.md:13]: Yeah the diff is wrong, I'll try again

Comment (caugustus-sourcegraph) [reply doc/admin/updates/pure_docker.md:13]: The diff still looks wrong - now it accidentally deletes the apache deployment (which is solely for the customer-replica branches), and deploy-caddy.sh is uncommented (not used for customer-replica branches). The pgsql ref looks wrong too.

**Review by caugustus-sourcegraph — APPROVED** [review]

**Pull request merged**